### PR TITLE
[37] About Me Fullstack Track

### PR DESCRIPTION
**Pull request by ToluEnny24**
aboutMe
I deleted the second .txt file and used the camelCase font

### CODE REVIEW

File: submissions/FabunmiTolulopeEniola/Mygoals.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+I am Fabunmi Tolulope Eniola. I am so excited to be a participant of Dufuna Codecamp and I hope at the end of the 6 months: I would be a full stack developer,  I would have participated in some hackathons and successfully built an amazing online portfolio.
```
Comment (TundeOladejo) [inline submissions/FabunmiTolulopeEniola/Mygoals.txt:1]: You are yet to delete this file from your submissions folder.  Delete this file from `submissions/FabunmiTolulopeEniola`, commit your changes and push.

File: submissions/FabunmiTolulopeEniola/aboutme.txt [hunk 2]
```diff
@@ -0,0 +1,5 @@
+I am Fabunmi Tolulope Eniola. I am 50% sanguine and 50% melancholic. This means I always have the top of the hour gist and alongside can be quiet almost simultaneously.
```
Comment (TundeOladejo) [inline submissions/FabunmiTolulopeEniola/aboutme.txt:1]: Delete this file also.

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

File: submissions/FabunmiTolulopeEniola/Mygoals.txt (hunk 1 above)
Comment (ToluEnny24) [reply submissions/FabunmiTolulopeEniola/Mygoals.txt:1]: I will do it immediately

Comment (adaugochi) [reply submissions/FabunmiTolulopeEniola/Mygoals.txt:1]: > Say for instance a Mentor comment of your PR, how do I go about it? How will the update be reflected on the Pull Request I already created? Below is a step to follow:

- Still, on that same branch, update your changes.

- Add your changes (git add .).

- Commit your changes (git commit  -m  "commit message").

- Finally, Push your change (git push origin branch_name)

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/FabunmiTolulopeEniola/Mygoals.txt (hunk 1 above)
Comment (ToluEnny24) [reply submissions/FabunmiTolulopeEniola/Mygoals.txt:1]: i have deleted the new branch

**Review by ToluEnny24 — COMMENTED** [review]
Thank you so much. i have effected the changes

File: submissions/FabunmiTolulopeEniola/Mygoals.txt (hunk 1 above)
Comment (ToluAkin) [inline submissions/FabunmiTolulopeEniola/Mygoals.txt:1]: @ToluEnny24 You need to this file on this branch.

**Review by ToluAkin — CHANGES_REQUESTED** [review]

**Review by TundeOladejo — APPROVED** [review]

**Review by adaugochi — APPROVED** [review]

**Pull request merged**